src/gurt/telemetry.c

```diff
@@ -386,26 +386,38 @@ d_tm_free_node(uint64_t *shmem_root, struct d_tm_node_t *node)
  *
  * \param[in]	val	Counter value
  * \param[in]	name	Counter name
+ * \param[in]	format	Output format.
+ *			Choose D_TM_VERBOSE for standard output.
```

Comment:
minor nit: As a casual reader of this docstring, it seems a little weird that VERBOSE is standard. Also, this seems to be mixing concepts a little. Usually verbosity refers to the volume of output rather than a specific format. It would strike me as more consistent to have D_TM_TEXT and D_TM_CSV or something like that. Not a blocker, just an observation.

Reply:
Right, I've changed the constant to D_TM_STANDARD per Kris's suggestion.  I used 'VERBOSE' originally because the volume of output was higher.  It used more words/characters than the CSV format.  At any rate, it has been changed in the next update.